test/conformance/enqueue/urEnqueueUSMMemAdvise.cpp

```diff
@@ -1,2 +1,60 @@
 // Copyright (C) 2023 Intel Corporation
 // SPDX-License-Identifier: MIT
+
+#include <uur/fixtures.h>
+
+using urEnqueueUSMMemAdviseWithParamTest =
+    uur::urUSMDeviceAllocTestWithParam<ur_mem_advice_t>;
+UUR_TEST_SUITE_P(urEnqueueUSMMemAdviseWithParamTest,
+                 ::testing::Values(UR_MEM_ADVICE_DEFAULT),
+                 uur::deviceTestWithParamPrinter<ur_mem_advice_t>);
+
+TEST_P(urEnqueueUSMMemAdviseWithParamTest, Success) {
+    ur_event_handle_t advise_event = nullptr;
+    ASSERT_SUCCESS(urEnqueueUSMMemAdvise(queue, ptr, sizeof(int),
+                                         getParam(), &advise_event));
+
+    ASSERT_NE(advise_event, nullptr);
+    ASSERT_SUCCESS(urQueueFlush(queue));
+    ASSERT_SUCCESS(urEventWait(1, &advise_event));
+
+    ur_event_status_t advise_event_status{};
+    ASSERT_SUCCESS(urEventGetInfo(
+        advise_event, UR_EVENT_INFO_COMMAND_EXECUTION_STATUS,
+        sizeof(ur_event_status_t), &advise_event_status, nullptr));
+    EXPECT_EQ(advise_event_status, UR_EVENT_STATUS_COMPLETE);
```

Comment:
advise_event should be released before the test ends

Reply:
👍